ui/src/style/pdf-viewer.less

```diff
@@ -14,5 +14,12 @@
  * Hide default annotations to avoid clutter and confusion
  */
 .annotationLayer {
-  display: none;
+  display: inline;
 }
+
+/**
+  * Hide default annotation specifically for citations to avoid unnecessary jumps
+  */
+a[href^="#cite"] {
```

Comment:
@jocelynzsun can you make the selector here more specific? One possible format could be:

```css
.class-for-default-pdf.js-annotations a[href^="#cite"]
```

Where `.class-for-default-pdf.js-annotations` could be determined by looking at the annotations PDF.js makes using the web inspector.

Right now, this will match any link that is not to a citation, though we have other links in the document we need to preserve.